src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  function getSumOrError(a) {
```

Comment:
```suggestion
  function getSum(a) {
```
You should just describe what's the purpose of the function, you don't need to write that it can return an error